tests/python/relax/test_tvmscript_parser_op_nn.py

```diff
@@ -35,6 +35,24 @@ def _check(
         tvm.ir.assert_structural_equal(parsed, expect)
 
 
+def test_conv1d():
+    @R.function
+    def foo(
+        x: R.Tensor((2, 3, 228), "float32"), w: R.Tensor((16, 3, 5), "float32")
+    ) -> R.Tensor((2, 16, 224), "float16"):
+        gv: R.Tensor((2, 16, 224), "float16") = R.nn.conv1d(x, w, out_dtype="float16")
```

Comment:
Not a big problem, but it is not a common case to take fp32 as input and fp16 as output :-)

Reply:
right, would be good to change x and w to float16 in here

Reply:
Fixed
Thank you for the review!